.gitignore

```diff
@@ -1,4 +1,6 @@
 *.swp
 *.swo
-build
+*build*
```

Comment:
Remove these?

Reply:
Do you mean removing the stars around build or anything else?

Reply:
I'd just leave the .gitignore the way it is.

Reply:
> I'd just leave the .gitignore the way it is.

OK I'll revert it.